css/css-contain/container-queries/at-container-parsing.html

```diff
@@ -150,9 +158,11 @@
   test_query_invalid('(width: 100px), (height: 100px)');
   test_query_invalid('(width: 100px) and (height: 100px)');
   test_query_invalid('(width: 100px) or (height: 100px)');
-  test_query_invalid('not (width: 100px)');
   test_query_invalid('foo (width: 100px)');
 
+  test_rule_valid('name not (width <= 500px)');
+  test_rule_valid('not (width <= 500px)');
```

Comment:
We should probably have a test that checks if `not` is parsed as part of the condition and not the name (probably using a reftest or such).

Reply:
As it stands, this passes in Safari because it's detected as a name.